### DISCUSSION TITLE
Added: LDM Notes for Oct. 22nd and Oct 24th, 2018

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#1985] [General]
[C# Language Design Notes for Oct 22, 2018](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-10-22.md)

Discussion of records proposals:

1. [The existing old one](https://github.com/dotnet/csharplang/blob/master/proposals/records.md)
2. [The data classes proposal](https://github.com/dotnet/csharplang/pull/1667)

[C# Language Design Notes for Oct 24, 2018](https://github.com/dotnet/csharplang/blob/master/meetings/2018/LDM-2018-10-24.md)

- [Adding Nullable Reference Type features to Nullable Value Types](https://github.com/dotnet/csharplang/issues/1865)
- Open issues with pattern matching

**Comment by ymassad** [comment]
How likely is it that records will be in C# 8?

**Comment by gafter** [comment]
In my opinion (_and this is only my personal opinion_), given (1) the state of the design being opened up compared to where we left it when the previous proposal was written, (2) the late date for C# 8 work, and (3) many other things on our engineering plates competing for resources to complete C# 8 (including lots of work remaining on nullable reference types and default interface methods, as well as areas where new language features intersect) together imply that the likelihood of records making C# 8 is not very high.  I don't think this opinion is yet widely held.
